src/output.h

```diff
@@ -4,9 +4,11 @@
 #include "color.h"
 #include "cursesdef.h"
 #include "catacharset.h"
+#include "input.h"
```

Comment:
Why do you need this include? If it's just for the `input_context` as the function parameter, a forward declaration of it should sufficient (no include needed).